supertokens_python/recipe/passwordless/recipe_implementation.py

```diff
@@ -226,19 +228,21 @@ async def update_user(self, user_id: str,
             return UpdateUserEmailAlreadyExistsErrorResult()
         return UpdateUserPhoneNumberAlreadyExistsErrorResult()
 
-    async def delete_email_for_user(self, user_id: str, user_context: Dict[str, Any]) -> UpdateUserResult:
+    async def delete_email_for_user(self, user_id: str, user_context: Dict[str, Any]) -> DeleteUserInfoResult:
         user = await self.get_user_by_id(user_id, user_context)
         if user is None:
-            return UpdateUserUnknownUserIdErrorResult()
-
-        return await self.update_user(user_id, '', user.phone_number, user_context)
+            return DeleteUserInfoUnknownUserIdErrorResult()
+        data = {'userId': user_id, 'email': None, 'phoneNumber': user.phone_number}
+        await self.querier.send_put_request(NormalisedURLPath('/recipe/user'), data)
```

Comment:
The code should be like this.. (same goes for the other function). Please double check that the below is actually right.

```python
async def delete_email_for_user(self, user_id: str, user_context: Dict[str, Any]) -> DeleteUserInfoResult:
        data = {'userId': user_id, 'email': None}
        result = await self.querier.send_put_request(NormalisedURLPath('/recipe/user'), data)
        if result['status'] == 'OK':
           return DeleteUserInfoOkResult()
        return DeleteUserInfoUnknownUserIdErrorResult()
```